lib/network/modules/InteractionHandler.js

```diff
@@ -705,6 +710,30 @@ class InteractionHandler {
       this.body.emitter.emit('hidePopup');
     }
   }
+
+
+  /**
+   * Ensure zoom min/max values are acceptable
+   * If they aren't, remove them from `options`
+   * @private
+   */
+  _checkRemoveZoomMinMax(options) {
+    const { zoomMin, zoomMax } = options;
+
+    const isValid = value => (
+      util.isNumber(value)
+        && value >= this.defaultOptions.zoomMin
+        && value <= this.defaultOptions.zoomMax
+    );
+
+    if (!isValid(zoomMin) || zoomMin > zoomMax) {
+      delete options.zoomMin;
```

Comment:
It would be useful to say what's wrong. That is `console.error("There's X wrong with zoomMin.")`.